food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -51,155 +63,287 @@
+                  <text variable="publisher-place" prefix=", "/>
+                </group>
+                <text macro="Cite-and-URL"/>
+              </else-if>
+              <else-if type="paper-conference">
+                <!-- conference paper -->
+                <text variable="title"/>
+                <group>
+                  <text term="in" text-case="capitalize-first" font-style="italic"/>
+                  <text macro="editor" prefix=" "/>
+                </group>
+                <text variable="container-title" font-style="italic"/>
+                <group suffix=".">
+                  <label suffix=" " variable="page" form="short"/>
+                  <text variable="page"/>
+                </group>
+                <text variable="collection-title"/>
+                <group suffix=".">
+                  <text term="presented at"/>
+                  <text variable="event" prefix=" "/>
+                  <date form="text" variable="issued" prefix=", "/>
+                  <text variable="publisher-place" prefix=", "/>
+                </group>
+                <text macro="Available-at"/>
+              </else-if>
+              <else>
+                <!-- other item types -->
+                <text variable="title" suffix="."/>
+                <text macro="publisher-and-place" suffix="."/>
+                <text macro="Cite-and-URL"/>
+              </else>
             </choose>
           </group>
-        </else-if>
-        <else-if type="thesis" match="any">
-          <text variable="title" font-style="italic" suffix="."/>
-          <text macro="publisher"/>
-          <text variable="genre" prefix=" (" suffix=")"/>
-        </else-if>
-        <else-if type="paper-conference" match="any">
-          <text variable="title" font-style="italic" suffix=". "/>
-          <text value="Paper presented at"/>
-          <group delimiter=", " prefix=" " suffix=".">
-            <text variable="event"/>
-            <text variable="event-place"/>
-            <date form="text" date-parts="year-month-day" variable="issued"/>
-          </group>
-          <choose>
-            <if match="any" variable="URL">
-              <text macro="Available-at-and-accessed" prefix=" "/>
-            </if>
-          </choose>
-        </else-if>
+        </if>
         <else>
-          <text variable="title" prefix=" " suffix="."/>
-          <group prefix=" ">
-            <text variable="container-title" form="short" font-style="italic" suffix=", "/>
-            <text variable="volume" form="short"/>
-            <text variable="issue" form="short" prefix="(" suffix=")"/>
+          <!-- case -->
+          <group delimiter=" ">
+            <group>
+              <text variable="title"/>
+              <text variable="volume" prefix=", "/>
+            </group>
+            <text variable="container-title"/>
+            <text variable="page"/>
+            <text variable="authority" prefix="("/>
```

Comment:
OK, I see. It's supressing that because it's just a term, so you need to wrap it in a macro, i.e. create
```
<macro name="anonymous">
  <text term="anonymous" text-case="capitalize-first"/>
</macro>
```

and then use `<text macro="anonymous"/>` in substitute.

Your example, though, indicates a problem in your style: you presumably don't want anonymous for in-text citations of legal cases (as you explicitly don't have it in the bibliography).